ocs_ci/helpers/helpers.py

```diff
@@ -137,7 +137,7 @@ def create_pod(
     node_selector=None,
     command=None,
     command_args=None,
-    deploy_pod_status=constants.STATUS_COMPLETED,
+    deploy_pod_status=constants.STATUS_RUNNING,
```

Comment:
why did you change this status ?
does this change impact other test ?
if you need in your test, you can pass it as parameter when you call to this function.

Reply:
You couldn't pass and I've added it to pod_factory. Anyway I'm dropping it. Thanks